Initial attempt at scala automated comments for two-fer (#1148)
exercism/website-copy: automated-comments/scala/two-fer/no_default_param.md

> Try specifying a (default parameter value)[https://docs.scala-lang.org/tour/default-parameter-values.html] for the `name` parameter. Providing a default parameter will allow for a solution utilzing a single function without conditional logic.

Reviewer: ```suggestion
Specify a (default parameter value)[https://docs.scala-lang.org/tour/default-parameter-values.html] for the `name` parameter. Providing a default parameter removes the need for conditional logic.
```
